flow.yaml

```diff
@@ -0,0 +1,45 @@
+import:
+  - word-counts.flow.yaml
+
+captures:
+  acmeCo/postgres/source-documents:
+    endpoint:
+      airbyteSource:
+        image: ghcr.io/estuary/source-postgres:1d9247b
```

Comment:
Are there other `source-postgres` changes that haven't been merged yet, or could this be changed to `dev` without issues now?